src/components/button/button.test.tsx

```diff
@@ -1,12 +1,51 @@
 import React from 'react';
-import { render, screen } from '@testing-library/react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom';
 
 import Button from './button';
+import { ButtonProps } from './button';
+
+function renderAndGetButton(props: ButtonProps): HTMLElement {
+  render(<Button label={props.label || 'Button'} {...props} />);
```

Comment:
```suggestion
  render(<Button {...props} />);
```

You can do this when change above